src/inc/msquic.h

```diff
@@ -232,6 +232,27 @@ typedef enum QUIC_DATAGRAM_SEND_STATE {
 #define QUIC_DATAGRAM_SEND_STATE_IS_FINAL(State) \
     ((State) >= QUIC_DATAGRAM_SEND_LOST_DISCARDED)
 
+typedef enum QUIC_EXECUTION_CONFIG_FLAGS {
+    QUIC_EXECUTION_CONFIG_FLAG_NONE             = 0x0000,
+} QUIC_EXECUTION_CONFIG_FLAGS;
+
+DEFINE_ENUM_FLAG_OPERATORS(QUIC_EXECUTION_CONFIG_FLAGS)
+
+//
+// A custom configuration for thread execution in QUIC.
+//
+typedef struct QUIC_EXECUTION_CONFIG {
+
+    QUIC_EXECUTION_CONFIG_FLAGS Flags;
+    uint32_t SleepTimeoutUs;            // Number of microseconds of idleness before sleeping a datapath thread.
```

Comment:
PollingIdleTimeoutUs

Time before a polling thread, with no work to do, sleeps.